test/routes/forms/submission/submissionName/router.skiptest.ts

```diff
@@ -0,0 +1,77 @@
+// import { vi, describe, beforeAll, it, expect } from "vitest";
```

Comment:
The original implementation was not compatible because of authentication field required to be added by the middleware.  This led to the implementation of a different testing strategy which is only partially completed.  In an effort to not block the functionality from being reviewed and merged the file is only commented out temporarily and will be completed in the near future.